iothub/service/src/Common/TrackingHelper.cs

```diff
@@ -25,17 +33,32 @@ public static class TrackingHelper
         private const string PartitionPrefix = "-P:";
         private const string TimeStampPrefix = "-TimeStamp:";
 
+        /// <summary>
+        /// Generates a tracking Id. Not used.
```

Comment:
The method is "Not used"? This will show up in intellisense for one who wants to consume this method. Do we need the "Not used"?

Reply:
Same comment for GenerateTrackingId

Reply:
... and other methods below :)

Reply:
It will show up as having 0 references in intellisense (if unused), any reason why we want to call it out in comments as well?
Also, +1 to not having our docs say "Not used" :)

Just for reference: we had a bunch of unused properties in transport settings. I had called those out as "unused" in our docs, just so that users are aware that even if they set those properties, the sdk will not reference/ use it anywhere. (not ideal, but a workaround until we either implement them or get rid of them).

Reply:
I marked this as Obsolete and wanted to make it clear that this is not being used and should not be used. And ideally we would remove it after sometime as it is marked obsolete.